### ISSUE TITLE
Repository Out of Date

### ISSUE THREAD

**Issue by XedinUnknown** [easydigitaldownloads/EDD-License-handler#43]
Hi,

I am developing a plugin module that uses EDD for licensing. In particular, I use the `EDD_License` class. In order to be able to write tests for the module, I need to depend on a package that contains this class. This package seems to be the official package for that and other related classes. However, it is severely out of date. Would it be possible to please update this package to reflect the latest code?

### Update
So, to summarize what I would need in order to be able to write correct code that depends on the EDD Licensing API "SDK", if I may call it that:

1. Maintain an up-to-date reflection of the code.
2. Add package info compatible with Composer/Packagist, including license.
3. Publish the package on Packagist.

**Comment by XedinUnknown** [comment]
In the case you do not wish to update the code, I we would be happy to maintain that code in our own package. With your permission, of course.

**Comment by XedinUnknown** [comment]
Maybe related to #41.

**Comment by XedinUnknown** [comment]
Just realized that this code is not available on Packagist. Kindly include package info in the update, and publish it, if possible.

**Comment by XedinUnknown** [comment]
It's also unclear how this code is licensed. I would assume it's GPL, since it is made for WordPress, but it would be helpful if this was made explicit in the package.

**Comment by cklosowski** [comment]
@XedinUnknown The intended use of this repo was not as a way to build a license handler for software other than Easy Digital Downloads extensions. Over the years we've improved the license integrator to the point that this is only needed when building an EDD Add-on.

We've updated the README.MD to better explain how to use the `EDD_License` class. In short, it should not be included with your EDD Add-On if you are building one. The repo was not intended to be a 'template' for building a license handler, but was originally intended to be used for people integrating with Easy Digital Downloads.

Please check out the new readme file.

**Comment by XedinUnknown** [comment]
Hi,

My intention is not to ship it with my plugin. It was to use it as dev dependency, so that my project has access to the code which it invokes. This way, I could make it testable, and enable static analysis. Now that you've removed the files, even the manual package definition we had put in will not work. I understand this means that you will not be maintaining that repo.

Thank you.
